config/permissions.tmpl.yml

```diff
@@ -35,6 +35,7 @@
 - ems-type:gce_network
 - ems-type:hawkular
 - ems-type:hawkular_datawarehouse
+- ems-type:monitoring
```

Comment:
OK Thanks, turns out this was a mistake, and should only be done for the leaf classes:

https://github.com/ManageIQ/manageiq/pull/15506